### ISSUE TITLE
remove user_agent check in messenger webhook

### ISSUE THREAD

**Issue by adamjuhasz** [Bottr-js/Bottr#36]
I can't find any official documentation that the user agent will be `facebookplatform`.

Is checking the `x-hub-signature` enough, since that is the only official way of checking validity?

**Comment by jcampbell05** [comment]
@adamjuhasz I saw it once in one of the HTTP Requests from Facebook but `x-hub-signature` probably is enougth as I've seen cases where the `facebookplatform` isn't in the agent.

**Comment by adamjuhasz** [comment]
I was looking at the current way of handling webhooks, do I undestand it correctly that all webhooks will be at `/webhook` and an event is sent with the event name `'webhook'` to all listeners? Since `x-hub-signuture` is commonly used by webhook APIs (ex github), I would be scared of using just that to differentiate FB from other web hooks.

Is it better to se up multiple webhook paths, one for each service and add custom events?
````
this.router.post('/webhook/facebook', (..) => this.trigger('webhook-facebook', req, res))
this.router.post('/webhook/twillio',  (...) => this.trigger('webhook-twillio', req, res))
````

This could be parameterized with: 
````
this.router.post('/webhook/:platform',  (req, res, next) => {
    this.trigger(`webhook-${rew.params['platform']}`, req, res, next)
});
````

**Comment by jcampbell05** [comment]
@adamjuhasz  Yeah I totally agree, I would like for us to move towards something like that especially as we will have other endpoints for other features like pushing content to the Bot.

**Comment by jcampbell05** [comment]
We should be able to implement this now @vutran's ES6 classes are now implemented.

**Comment by vutran** [comment]
@jcampbell05 Actually, the ES6 classes are not merged (#41). There's still some broken tests I need to fix which should be done soon (got a lot of things going on right now).

**Comment by jcampbell05** [comment]
@vutran  let me know if I can help :) I think we should merge into `master` and split it up :) there are a lot of things that need to come together before 0.2.0
